splice_machine/src/main/java/com/splicemachine/derby/utils/Vacuum.java

```diff
@@ -130,8 +159,16 @@ public void vacuumDatabase(long oldestActiveTransaction) throws SQLException{
                             requiresDroppedId = true;
                         }
                     }
```

Comment:
What is the differences between transactionId and droppedTransactionId? I cannot find usage of droppedTransactionId.

Reply:
transactionId is the Id of the transaction that created the table, droppedTransactionId is the id of the txn that dropped it.